main

```diff
@@ -0,0 +1,46 @@
+require_relative 'Teacher'
+require_relative 'Student'
+require_relative 'Corrector'
+require_relative 'Classroom'
+require_relative 'Book'
+require_relative 'Rental'
```

Comment:
- [ ] I was unable to run the app at first because of the naming of the required files.

![image](https://user-images.githubusercontent.com/16058470/136643991-fb2794b3-cdd8-40a1-a3d1-7e132121e373.png)

Note that you're requiring a file, not the class and as such, the first letter should also be in lower case. If a file name is `myfile.rb` with class `Myfile`, in the main file or wherever you need it, it should be `require_relative 'myfile'`, not `require_relative 'Myfile'`.

Kindly make the changes to all applicable files.